README.md

```diff
@@ -7,25 +7,39 @@ This dbt package contains models, macros, seeds, and tests for [FullStory](https
 | sessions | Session-level aggregations, including event counts broken down by type, location and device information, duration, FullStory session replay links, etc.
 | users | User-level aggregations, including email addresses, location and device information, session counts, etc.
 
+## Vars
```

Comment:
This reminds me, at some point, we should open a wiki for this package so that we don't need to PR readme changes.